orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/services/AuditServiceTest.java

```diff
@@ -138,4 +143,37 @@ void shouldAddCountryCodeExtensionToPhoneNumberEvents() {
 
         assertThat(extensions, hasFieldWithValue("phone_number_country_code", equalTo("44")));
     }
+
+    @Test
+    void TxmaHeaderShouldBeAddedToAuditEvent() throws JOSEException, InvalidEncodingException {
+        var auditService = new AuditService(FIXED_CLOCK, configurationService, awsSqsClient);
+
+        attachAuditField(TXMA_ENCODED_HEADER, TXMA_ENCODED_HEADER_VALUE);
+        auditService.submitAuditEvent(
+                TEST_EVENT_ONE,
+                "request-id",
+                "session-id",
+                "client-id",
+                "subject-id",
+                "email",
+                "ip-address",
+                "07700900000",
+                "persistent-session-id",
+                pair("key", "value"),
+                pair("key2", "value2"));
```

Comment:
If we don't care about any of these values, this line could be `auditService.submitAuditEvent(TEST_EVENT_ONE, "client-id", TxmaAuditUser.user())` as a one-liner.